Tests/test_parameter.py

```
@@ -1,29 +1,109 @@
+        param = Parameter(AttrType.INPUT, AInteger)
+        attr_id = self._make_attr(param)
+
+        assert attr_id in dpg.get_all_items()
+        assert param.get_value(attr_id) == 0
+
+    def test_Parameter_incompatible_parent_raises(self):
+        # build должен поднять Exception, если родитель — не mvNode
+        param = Parameter(AttrType.INPUT, AInteger)
+
+        with dpg.window() as window_id:
+            with self.assertRaises(Exception):
```

Ruff (0.15.15)

[warning] Do not assert blind exception: `Exception`

(B017)